_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Prometheus SQL and Metrics Analytics Observability plugin
+nav_order: 40
+redirect_from:
+  - /observing-your-data/prometheusmetrics/
+---
+
+# Prometheus SQL and Metrics Analytics Observability plugin
+Introduced 2.4
+{: .label .label-purple }
+
+Starting with OpenSearch 2.4, you can use two new additional features within the **Observability** application in OpenSearch Dashboards: the Metrics UI for visualization, and connecting to Prometheus to gather metrics.
+
+![Prometheus and Metrics]({{site.url}}{{site.baseurl}}/images/metrics/metricsgif.gif)
+
+## Metrics Analytics UI
```

Comment:
I think we should focus less on the UI and talk about what the user can do with the Prometheus connection. In other words, I'm not sure showing this adds any value.

Reply:
the metics section here also has new features so I do think these sections add value

Reply:
What new features? For accessibility we need to describe what we are demonstrating in this image. If it's just to point out that there is a new UI, I don't think we need it. We just need to make sure we are telling the users how to perform their task using the new UI.

Reply:
ok, let me look and see if I can re-work it

Reply:
made a revision